README.md

```diff
@@ -44,8 +44,24 @@ dependency. ``rtree`` requires the C library [``libspatialindex``](https://githu
 
 **Install**
 
-Then, installation works as normal: ``pip install geopandas``
+GeoPandas depends on several low-level libraries for geospatial analysis. Depending on the system and package
+manager that you use, this may cause dependency conflicts if you are not careful. See below for suggestoins:
 
+If you are installing on a **linux** system, installation should be straightforward:
+
+``pip install geopandas``
+
+If you are installing on **OSX** or **Windows**, we recommend installing with the `anaconda` distribution using
```

Comment:
I would not necessarily recommend full anaconda, eg I use only conda through miniconda (although that might make it more confusing for newcomers?